cookiecutter-frontend-app/cookiecutter.json

```diff
@@ -0,0 +1,8 @@
+{
+  "repo_owner": "edx",
```

Comment:
This allows community members to use this template without _also_ search/replacing for these instances, especially since there are many false positive matches.